components/apps/ptp-operator/README.md

```diff
@@ -0,0 +1,32 @@
+# OpenShift Precision Time Protocol Operator
+
+Installs the OpenShift PTP Operator.
+
+Do not use the `base` directory directly, as you will need to patch the `channel` based on the version of OpenShift you are using, or the version of the operator you want to use.
```

Comment:
I thought with 4.10 we moved to a "stable" channel distinction, versus a version number that has to increment every minor release?

Reply:
Agree in that "stable" is going to be channel no matter the OCP release. However in the future we might be interested on getting some preview, fast channel, if needed. Gonna correct the README.md.

Reply:
+1